roles/installer/defaults/main.yml

```diff
@@ -286,6 +286,8 @@ web_command: []
 rsyslog_args:
   - /usr/bin/launch_awx_rsyslog.sh
 rsyslog_command: []
+redis_args: ''
+redis_command: []
```

Comment:
It would be good to add docs in the README.md for these as well with an example use case.  But this is a niche case, so not strictly required for merge.